lib/devise/models/rememberable.rb

```diff
@@ -100,6 +100,10 @@ def rememberable_options
       def after_remembered
       end
 
+      def remember_expired?
+        remember_created_at.nil? || (remember_expires_at <= Time.now.utc)
```

Comment:
Shouldn't we consider app timezone?

```ruby
Time.zone.now
```

Reply:
@duderamos yes, makes sense. Making this change.